cryptarithm.py

```diff
@@ -138,7 +138,7 @@ def build_dqm(variable_list: List[LetterVariable], coefficient_map: dict) -> DQM
     response = LeapHybridDQMSampler(
         solver="hybrid_discrete_quadratic_model_version1", auto_scale=False
```

Comment:
I believe I've been told this before and just reconfirmed for myself that `auto_scale` does nothing when passed as a keyword arg here. Would like to be double checked on that again but I think we can remove this here if in scope.

Reply:
Sure, I can remove it